edurange_refactored/flask/modules/utils/scenario_interface.py

```diff
@@ -0,0 +1,136 @@
+
+from celery import group
+from edurange_refactored.extensions import db
+from edurange_refactored.user.models import User, GroupUsers, StudentGroups, Scenarios
+from edurange_refactored.notification_utils import NotifyCapture
+from edurange_refactored.tasks_sister import (
+    create_scenario_task, 
+    start_scenario_task, 
+    stop_scenario_task, 
+    update_scenario_task,
+    destroy_scenario_task
+    )
+from edurange_refactored.tasks import (
+    CreateScenarioTask
+    )
+from edurange_refactored.scenario_utils import gen_chat_names
+from flask import jsonify, abort, g
+
+def scenario_create(scenario_type, scenario_name, studentGroup_name):
+    db_ses = db.session
+    owner_user_id = g.current_user_id
+    print("CREATE REQ INFO: ",scenario_type, scenario_name, studentGroup_name)
+    students = (
+        db_ses.query(User.username)
+        .filter(StudentGroups.name == studentGroup_name)
+        .filter(StudentGroups.id == GroupUsers.group_id)
+        .filter(GroupUsers.user_id == User.id)
+        .all()
+    )
+    for i, s in enumerate(students):
+        students[i] = s._asdict()
+
+    Scenarios.create(name=scenario_name, description=scenario_type, owner_id=owner_user_id)
+    NotifyCapture(f"Scenario {scenario_name} has been created.")
+    
+    scenario_id = db_ses.query(Scenarios.id).filter(Scenarios.name == scenario_name).first()
+    
+    s_id_list = list(scenario_id._asdict().values())[0]
+
+
+    scenario = Scenarios.query.filter_by(id=s_id_list).first()
+    scenario.update(status=7)
+    
+    group_id = db_ses.query(StudentGroups.id).filter(StudentGroups.name == studentGroup_name).first()
+    group_id = group_id._asdict()
+    
+    
+    student_ids = db_ses.query(GroupUsers.id).filter(GroupUsers.group_id == group_id['id']).all()
+    namedict = gen_chat_names(student_ids, scenario_id)
+
+    if ( scenario_name is None \
```

Comment:
scenarios cant have duplicate names; add validation for lack of symbols, can't start with a number